src/node_quic_session.h

```diff
@@ -221,266 +232,269 @@ class QuicSession : public AsyncWrap {
-    void* user_data);
+      ngtcp2_conn* conn,
+      uint64_t offset,
+      size_t datalen,
+      void* user_data);
   static int OnAckedStreamDataOffset(
-    ngtcp2_conn* conn,
-    int64_t stream_id,
-    uint64_t offset,
-    size_t datalen,
-    void* user_data,
-    void* stream_user_data);
+      ngtcp2_conn* conn,
+      int64_t stream_id,
+      uint64_t offset,
+      size_t datalen,
+      void* user_data,
+      void* stream_user_data);
   static int OnStreamClose(
-    ngtcp2_conn* conn,
-    int64_t stream_id,
-    uint16_t app_error_code,
-    void* user_data,
-    void* stream_user_data);
+      ngtcp2_conn* conn,
+      int64_t stream_id,
+      uint16_t app_error_code,
+      void* user_data,
+      void* stream_user_data);
   static int OnStreamOpen(
-    ngtcp2_conn* conn,
-    int64_t stream_id,
-    void* user_data);
+      ngtcp2_conn* conn,
+      int64_t stream_id,
+      void* user_data);
   static int OnStreamReset(
-    ngtcp2_conn* conn,
-    int64_t stream_id,
-    uint64_t final_size,
-    uint16_t app_error_code,
-    void* user_data,
-    void* stream_user_data);
+      ngtcp2_conn* conn,
+      int64_t stream_id,
+      uint64_t final_size,
+      uint16_t app_error_code,
+      void* user_data,
+      void* stream_user_data);
   static int OnRand(
-    ngtcp2_conn* conn,
-    uint8_t* dest,
-    size_t destlen,
-    ngtcp2_rand_ctx ctx,
-    void* user_data);
+      ngtcp2_conn* conn,
+      uint8_t* dest,
+      size_t destlen,
+      ngtcp2_rand_ctx ctx,
+      void* user_data);
   static int OnGetNewConnectionID(
-    ngtcp2_conn* conn,
-    ngtcp2_cid* cid,
-    uint8_t* token,
-    size_t cidlen,
-    void* user_data);
+      ngtcp2_conn* conn,
+      ngtcp2_cid* cid,
+      uint8_t* token,
+      size_t cidlen,
+      void* user_data);
   static int OnRemoveConnectionID(
-    ngtcp2_conn* conn,
-    const ngtcp2_cid* cid,
-    void* user_data);
+      ngtcp2_conn* conn,
+      const ngtcp2_cid* cid,
+      void* user_data);
   static int OnUpdateKey(
-    ngtcp2_conn* conn,
-    void* user_data);
+      ngtcp2_conn* conn,
+      void* user_data);
   static int OnPathValidation(
-    ngtcp2_conn* conn,
-    const ngtcp2_path* path,
-    ngtcp2_path_validation_result res,
-    void* user_data);
+      ngtcp2_conn* conn,
+      const ngtcp2_path* path,
+      ngtcp2_path_validation_result res,
+      void* user_data);
   static void OnIdleTimeout(
-    uv_timer_t* timer);
+      uv_timer_t* timer);
   static void OnRetransmitTimeout(
-    uv_timer_t* timer);
+      uv_timer_t* timer);
   static int OnExtendMaxStreamsUni(
-    ngtcp2_conn* conn,
-    uint64_t max_streams,
-    void* user_data);
+      ngtcp2_conn* conn,
+      uint64_t max_streams,
+      void* user_data);
   static int OnExtendMaxStreamsBidi(
-    ngtcp2_conn* conn,
-    uint64_t max_streams,
-    void* user_data);
+      ngtcp2_conn* conn,
+      uint64_t max_streams,
+      void* user_data);
   static int OnExtendMaxRemoteStreamsUni(
```

Comment:
Seems not implemented. Also `OnExtendMaxRemoteStreamsBidi`

Reply:
Will handle these later on

Reply:
Actually... nevermind... can pull these out now